module_utils/network_lsr/argument_validator.py

```diff
@@ -896,6 +896,16 @@ def _validate_post_fields(self, value, name, result):
         state = result.get("state")
         persistent_state = result.get("persistent_state")
 
+        # if persistent state is present state cannot be set to present or absent
+        if persistent_state in result.keys() and state in result.keys():
```

Comment:
This does not seem to be right, on first glance it seems you need to check wether persistent_state is not `None`.